src/test/java/spark/InitExceptionHandlerTest.java

```diff
@@ -16,18 +16,16 @@
 
     @BeforeClass
     public static void setUpClass() throws Exception {
-
         service1 = ignite();
         service1.port(1122);
         service1.init();
         service1.awaitInitialization();
 
         service2 = ignite();
-        service2.port(1122);
+        service2.port(70000);
```

Comment:
Starting 2 servers on the same port is supported in some environments.
This test fails on my environment, as its allowed to have 2 servers on the same listening port.